integrations/external_ip.go

```diff
@@ -0,0 +1,22 @@
+package integrations
+
+import (
+	"fmt"
+	"net"
+)
+
+const NoExtIPApi = "no external ip address API available"
+
+// GetExternalIP
```

Comment:
Codacy found an issue: [comment on exported function GetExternalIP should be of the form "GetExternalIP ..."](https://app.codacy.com/gh/go-go-dojo/friendly-potato/pullRequest?prid=6165294)